Translators/WZDx/tim_generator.py

```diff
@@ -27,8 +29,10 @@ def get_anchor(feature):
     coords = feature["geometry"]["coordinates"]
     route = geospatial_service.point_to_route_id(
         feature["geometry"]["coordinates"][0][0], feature["geometry"]["coordinates"][0][1])
-    print('found route ' + route)
     # route = translateRoute(feature["properties"]["core_details"]["road_names"])
+    if route is None:
+        return None
+    print('found route ' + route)

```

Comment:
Rather than print, let's import logging and use that so we can control the level of logging.